components/src/organisms/footer/Footer.js

```diff
@@ -16,48 +17,94 @@ export default {
     AList,
     AListItem,
     ACookieMessage,
-    ADropdownList,
+    ACollapse,
     ANewsletter
   },
   props: {
+    /**
+     * Copyright text
+     */
     copyrightText: {
       type: String,
-      required: true
+      default: null
     },
+    /**
+     * Title for social media list
+     */
     socialListSubtitle: {
       type: String,
-      required: true
+      default: null
     },
+    /**
+     * Array with social media
+     */
     socialMenu: {
       type: Array,
-      required: true
+      default: null
     },
+    /**
+     * Array with menu
+     */
     menu: {
       type: Array,
-      required: true
+      default: null
     },
+    /**
+     * Cookie text
+     */
     cookieText: {
       type: String,
-      required: true
+      default: null
     },
+    /**
+     * Heading
+     */
     heading: {
       type: String,
-      required: true
+      default: null
     },
+    /**
+     * Input object
+     */
     input: {
       type: Object,
-      required: true
+      default: null
     },
+    /**
+     * Array with checkboxes
+     */
     checkboxes: {
       type: Array,
-      required: true
+      default: null
+    }
+  },
+  data () {
+    return {
+      windowWidth: window.innerWidth,
+      isLargeScreen: window.innerWidth > 768
+    }
+  },
+  watch: {
+    windowWidth (newWidth) {
+      this.isLargeScreen = newWidth > 768
     }
   },
+  mounted () {
+    this.$nextTick(() => {
+      window.addEventListener('resize', () => {
```

Comment:
It's not good for performance, please consider using [debounce](http://bencentra.com/code/2015/02/27/optimizing-window-resize.html) or matchMedia (Robert told us last time matchmedia is optimized but it needs to be checked).
Same in the watcher above